admin_manual/configuration_server/occ_command.rst

```diff
@@ -1100,6 +1100,67 @@ Remove a certificate::
 
 .. _trashbin_label:
 
+Status
+------
+
+Use the status command to retrieve information about the current installation::
+
+ $ sudo -u www-data php occ status
+   - installed: true
+   - version: 25.0.2.3
+   - versionstring: 25.0.2
+   - edition:
+   - maintenance: false
+   - needsDbUpgrade: false
+   - productname: Nextcloud
+   - extendedSupport: false
+
+This information can also be formatted via JSON instead of plain text::
+
+ $ php occ status --output=json_pretty
+ {
+     "installed": true,
+     "version": "25.0.2.3",
+     "versionstring": "25.0.2",
+     "edition": "",
+     "maintenance": false,
+     "needsDbUpgrade": false,
+     "productname": "Nextcloud",
+     "extendedSupport": false
+ }
+
+And finally, the ``-e`` (for exit code) parameter can be used to check
```

Comment:
I wouldn't mind a heading above this so we can link to it 🤔

Reply:
I've added a heading, is this the way you'd like it?